core/types/src/attributes.rs

```diff
@@ -23,33 +24,31 @@ impl Attributes {
         self
     }
 
-    /// Set the `transform` for the attributes
+    /// Set the extended features request mask (xfrm)
```

Comment:
renamed once I was able to track down a definition on [SGX 101](https://sgx101.gitbook.io/sgx101/sgx-bootstrap/enclave#sgx-enclave-attributes)

Reply:
Also not that this and `flags` should be using a bitflag style enum per the [developer reference](https://download.01.org/intel-sgx/sgx-linux/2.17.1/docs/Intel_SGX_Developer_Reference_Linux_2.17.1_Open_Source.pdf), but wanted to minimize this specific change